sklearn/tree/tree.py

```diff
@@ -1391,6 +1391,9 @@ class ExtraTreeClassifier(DecisionTreeClassifier):
 
     Attributes
     ----------
+    base_estimator_ : ExtraTreeClassifier
```

Comment:
I'm confused by these though. Aren't these from ExtraTreeSClassifier?

Reply:
Fixed! Thanks for pointing out.